kie-dmn/kie-dmn-core/src/test/java/org/kie/dmn/core/stronglytyped/JavadocTest.java

```diff
@@ -56,7 +57,7 @@ public JavadocTest(VariantTestConf testConfig) {
 
     @Test
     public void testDateAndTime() throws Exception {
-        final DMNRuntime runtime = createRuntime("0007-date-time.dmn", getClass());
+        final DMNRuntime runtime = createRuntime("0007-date-time.dmn", DMNStronglyTypedSupportTest.class);
```

Comment:
```suggestion
        final DMNRuntime runtime = createRuntime("0007-date-time.dmn", DMNRuntimeTest.class.class);
```

just personal preference since this was historically the test which used this model first :)

Reply:
Sure, thx!